cpu/cc2538/vectors.c

```diff
@@ -71,74 +71,74 @@ WEAK_DEFAULT void isr_dmaerr(void);
 /* interrupt vector table */
 ISR_VECTORS const void *interrupt_vector[] = {
     /* Exception stack pointer */
-    (void*) (&_estack),             /* pointer to the top of the stack */
+    __extension__ (void*) (&_estack),             /* pointer to the top of the stack */
```

Comment:
I played with this some time ago and the array can be rewritten as some struct with the stack pointer as first member and the function pointers as rest.

I'm against using "__extension__". Either we aim for "-pedantic -Werror" compilation, or we accept that we're violating the standard for convenience reasons.